e2e-tests/tfm/cypress/e2e/journeys/deal-cancellation/check-details.spec.js

```diff
@@ -50,7 +50,7 @@ context('Deal cancellation - check details', () => {
 
       cy.url().should('eq', relative(`/case/${dealId}/cancellation/effective-from-date`));
 
-      cy.completeDateFormFields({ idPrefix: 'effective-from-date', date: tomorrow.date });
+      cy.completeDateFormFields({ idPrefix: 'effective-from-date', date: yesterday.date });
```

Comment:
What's the reason for changing from yesterday to tomorrow please?

Reply:
The future date hasnt been implemented yet & will need a separate e2e test